<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
border-radius: 25px;
color: rgb(72, 61, 65);</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(72, 61, 65);</string>
    </property>
    <property name="text">
     <string>Strategy Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="strategy_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);
border-radius: 15px;
color: rgb(72, 61, 65);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers|Qt::ImhPreferUppercase</set>
    </property>
    <property name="inputMask">
     <string extracomment="Only numbers are allowed"/>
    </property>
    <property name="text">
     <string>MyGamble</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>8</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
border-radius: 25px;
color: rgb(72, 61, 65);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(72, 61, 65);</string>
    </property>
    <property name="text">
     <string>Order Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="order_size">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);
border-radius: 15px;
color: rgb(72, 61, 65);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers|Qt::ImhPreferUppercase</set>
    </property>
    <property name="inputMask">
     <string extracomment="Only numbers are allowed">0000</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="commence_trading">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-image: url(:/stripes/bg/stripes-light.png);
	color: rgb(72, 61, 65);
	border-radius: 15px;
border-width: 2px;
border-style: solid;
	border-color: rgb(72, 61, 65);
}
</string>
   </property>
   <property name="text">
    <string>Commence Trading</string>
   </property>
   <property name="icon">
    <iconset resource="ui_assets/resources.qrc">
     <normaloff>:/icons/order.png</normaloff>:/icons/order.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
border-radius: 25px;
color: rgb(72, 61, 65);</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(72, 61, 65);</string>
    </property>
    <property name="text">
     <string>Short Die Face</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="long_window">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);
border-radius: 15px;
color: rgb(72, 61, 65);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers|Qt::ImhPreferUppercase</set>
    </property>
    <property name="inputMask">
     <string extracomment="Only numbers are allowed">0000</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);</string>
   </property>
   <widget class="QComboBox" name="ticker_combo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
border-radius: 25px;
color: rgb(72, 61, 65);</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(72, 61, 65);</string>
    </property>
    <property name="text">
     <string>Long Die Face</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="short_window">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);
border-radius: 15px;
color: rgb(72, 61, 65);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers|Qt::ImhPreferUppercase</set>
    </property>
    <property name="inputMask">
     <string extracomment="Only numbers are allowed">0000</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
border-radius: 25px;
color: rgb(72, 61, 65);</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(72, 61, 65);</string>
    </property>
    <property name="text">
     <string>Jackpot Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="window">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/stripes/bg/stripes-light.png);
border-radius: 15px;
color: rgb(72, 61, 65);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers|Qt::ImhPreferUppercase</set>
    </property>
    <property name="inputMask">
     <string extracomment="Only numbers are allowed">0000</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ui_assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
